src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/database/jpa/entity/induction/WorkOnReleaseEntity.kt

```diff
@@ -117,23 +108,16 @@ enum class HopingToWork {
   NOT_SURE,
 }
 
-enum class NotHopingToWorkReason {
-  LIMIT_THEIR_ABILITY,
-  FULL_TIME_CARER,
+enum class AffectAbilityToWork {
```

Comment:
As well as merging the 2 enums, we are renaming a few (correcting the spelling of "offence" 😁 , and making the previous "health", "health_issues" and "no_reason" more verbose and explicit)
The rename of existing values in the database is covered in the flyway script below 👇